backend/app/views.py

```diff
@@ -139,6 +139,17 @@ def get_photographer(request, photographer_number=None):
     serializer = PhotographerSerializer(photographer_obj, many=photographer_number is None)
     return Response(serializer.data)
 
+@api_view(['GET'])
+def get_photographer_photos(request, photographer_number):
+    """
+    API endpoint to get a photographer based on the photographer_id
+    If given photographer_number, GETs associated photographs
+    """
+    photo_obj = Photo.objects.filter(photographer__number = photographer_number)
+    print(photo_obj)
+    serializer = PhotoSerializer(photo_obj, many=photographer_number is None)
```

Comment:
This validation is a little bit difficult to read, and that's a code smell that something is wrong. If photographer_number is None, then it doesn't make sense to return anything from this function, and it should probably send an error code that the frontend can interpret and display in some sensible way. You'd do this by checking whether `photographer_number is None` right at the top of this function, rather than inside of this serializer.